pages/_app.tsx

```diff
@@ -6,7 +6,8 @@ import { MDXProvider } from '@mdx-js/react'
 import { CssBaseline, GeistProvider, useTheme, GeistUIThemes, Image } from 'components'
 import ConfigContext from 'lib/config-provider'
 import useDomClean from 'lib/use-dom-clean'
-import { HybridCode, HybridLink, Layout, Search } from 'lib/components'
+import { HybridCode, HybridLink, Search } from 'lib/components'
+import Menu from 'lib/components/layout/menu'
```

Comment:
If you can load `Menu` and `Side` dynamically, it should help with the size of the HTML.

Reply:
Well, `Menu` seems to be dynamically loaded already.